index.html

```diff
@@ -7,7 +7,7 @@
 <head>
 	
 	<title>	
-		Paul Robert English
+		Paul English
 	</title>
 	
 	<meta name="viewport" content="width=device-width, initial-scale=1.0">
```

Comment:
needs closing characters />, also is initial scale supposed to be in the quotes? some help would be appreciated. thanks

Reply:
awesome thanks!